src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java

```diff
@@ -41,12 +41,12 @@
 import jdk.internal.foreign.abi.x64.windows.Windowsx64Linker;
 import jdk.internal.foreign.layout.AbstractLayout;
 
-import java.lang.foreign.GroupLayout;
-import java.lang.foreign.MemoryLayout;
-import java.lang.foreign.SegmentScope;
 import java.lang.foreign.FunctionDescriptor;
+import java.lang.foreign.GroupLayout;
 import java.lang.foreign.Linker;
+import java.lang.foreign.MemoryLayout;
 import java.lang.foreign.MemorySegment;
+import java.lang.foreign.SegmentScope;
 import java.lang.foreign.SequenceLayout;
```

Comment:
Please don't reorder these imports relative to upstream.

Reply:
Restored as requested above.